Marlin/example_configurations/Geeetech/GT2560/Configuration.h

```diff
@@ -1735,7 +1735,7 @@
 /**
  * Customize common displays for GT2560
  */
-#if ENABLED(ULTIMAKERCONTROLLER) || ENABLED(REPRAP_DISCOUNT_SMART_CONTROLLER) || ENABLED(G3D_PANEL)
+#if ENABLED(ULTIMAKERCONTROLLER) || ENABLED(REPRAP_DISCOUNT_SMART_CONTROLLER) || ENABLED(G3D_PANEL) || ENABLED(MKS_MINI_12864)
   #define SDSUPPORT   // Force SD Card support on for these displays
```

Comment:
Note that this makes it harder for users to disable `SDSUPPORT` for debugging, which was the intent of having it as a separate option, disabled by default. In the long run, we may set a flag for displays that have SD built-in and throw out a warning to users who forget to enable `SDSUPPORT` on those displays.